### PR TITLE
Minor fixes in latest sections

### PR DESCRIPTION
**Pull request by saraedum**

### CODE REVIEW

File: flatsurf_algorithms.tex [hunk 1]
```diff
@@ -828,7 +828,7 @@ \subsection{$\GL(2,\bR)$-action and linear manifolds}
 
 \begin{proof}
 Just use the fact that the real projection $x+iy \mapsto x$ and imaginary projection
-$x+iy \mapsto y$ live the tangent space invariant (since they are limit of $\GL(2,\bR)$
```
Comment (saraedum) [inline flatsurf_algorithms.tex:831]: What did you mean to write here?

**Comment by saraedum** [comment]
@videlec please have a look. I am not convinced that I got this all right.

File: flatsurf_algorithms.tex (hunk 1 above)
Comment (videlec) [reply flatsurf_algorithms.tex:831]: You are not allowed to apply an element of `Mat(2, R)` to a translation surface but only elements in `GL(2, R)`. However, infinitesimally (in the tangent space) that makes perfect sense. The way to proceed is to write down the (derivative) of the action on the tangent bundle. It is just the simple formula I wrote somewhere. And it is clear that this action extends continuously to `Mat(2, R)`.

File: flatsurf_algorithms.tex [hunk 2]
```diff
@@ -766,17 +766,17 @@ \subsection{The tangent space to a translation surface}
 \begin{lemma}
 Let $T$ be a combinatorial triangulation with $d$ edges. The sum of triangle
 relations is zero and it is the only relation between them. In particular,
-$V(T)$ has codimension $d-1$.
+$V(T)$ has codimension $\textrm{faces}-1$.
```
Comment (videlec) [inline flatsurf_algorithms.tex]: I agree that $d$ is the number of edges. But `$\textrm{faces}$` is not so nice for the number of faces. At some point, I thought about using `n_v`, `n_e` and `n_f` for number of vertices, edges and faces.

**Review by videlec — COMMENTED** [review]
I agree with all your changes!

File: flatsurf_algorithms.tex (hunk 1 above)
Comment (saraedum) [reply flatsurf_algorithms.tex:831]: Sorry, the grammar just doesn't work out in the original phrase. Did you mean "live in" or "lift"?

File: flatsurf_algorithms.tex (hunk 2 above)
Comment (saraedum) [reply flatsurf_algorithms.tex]: Ok. I'll fix it.

File: flatsurf_algorithms.tex (hunk 1 above)
Comment (slel) [reply flatsurf_algorithms.tex:831]: "leave the tangent space invariant"

Comment (saraedum) [reply flatsurf_algorithms.tex:831]: lol, that makes a lot of sense. I thought of "the invariant of the tangent space" (whatever that might have been) and could not make any sense of it.

Comment (saraedum) [reply flatsurf_algorithms.tex:831]: Fixed. Thanks @slel.

**Pull request merged**